packages/react/src/functions/safe-dynamic-require.ts

```diff
@@ -5,6 +5,5 @@ import { Builder } from '@builder.io/sdk';
 const noop = () => null;
 // Allow us to require things dynamically safe from webpack bundling
 
-export let safeDynamicRequire: typeof require;
+export let safeDynamicRequire: typeof require = noop as any;
 if (typeof globalThis.require === 'function') safeDynamicRequire = eval('require');
-safeDynamicRequire ??= noop as any;
```

Comment:
Your change here will impact the behavior of how `safeDynamicRequire` is set. I would prefer to rollback as it is unrelated to this PR

more info on this operator: https://www.joshwcomeau.com/operator-lookup?match=logical-nullish-assignment

Reply:
does the new change look good?